src/TurbulenceClosures/turbulence_closure_implementations/ri_based_vertical_diffusivity.jl

```diff
@@ -1,4 +1,4 @@
-using Oceananigans.Architectures: architecture, on_architecture
```

Comment:
all these `on_architecture` changes create some noise in this PR which is making it harder to review, might make sense to put them in another PR